cohorts/cohort.py

```diff
@@ -470,23 +489,117 @@ def load_variants(self, patients=None, filter_fn=None):
             Dictionary of patient_id to VariantCollection
         """
         filter_fn = first_not_none_param([filter_fn, self.filter_fn], no_filter)
+        if filter_fn is None:
```

Comment:
Actually, this seems to happen 4+ times, so I'd extract into a function as I mentioned again below